README.md

```diff
@@ -37,26 +39,33 @@ var_dump($aggregator->getMergedConfig());
 Result:
 
 ```php
-array(3) {
+array(5) {
   'parameter_usage' =>
   string(3) "bar"
   'parameter_name' =>
   string(5) "%foo%"
   'recursive_parameter_usage' =>
   string(3) "qoo"
+  'parameterized_parameter_usage' =>
+  string(3) "qoo"
```

Comment:
Is this correct? Based on the examples, it seems like it would become "bar" (bar.quux resolves to parameter "foo", which resolves to the value "bar")?

Reply:
Nope, you're right. I've written this by myself instead of executing the script instead.
Fixed that in the latest commits.